docs/standard/security/vulnerabilities-cbc-mode.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Timing vulnerabilities with CBC-mode symmetric decryption using padding"
+title: Detect timing vulnerabilities in CBC-mode symmetric decryption
```

Comment:
Or maybe just "CBC decryption vulnerability"?

Reply:
Yeah, that makes it all fit. I'll take that.